src/doctypes/customizers.js

```diff
@@ -0,0 +1,176 @@
+import mergeWith from 'lodash/mergeWith'
+import mapValues from 'lodash/mapValues'
+
+// We trust the server by default
+const DEFAULT_POLICY = 'server'
+
+// When the value of an array is present on the server but not on
```

Comment:
« is present on the server document ». We implicitly know that we are talking about documents, but explicit is better